### PR TITLE
add centosstream-8

### PR DESCRIPTION
**Pull request by bryceml**
### What does this PR do?
adds centosstream-8

### CODE REVIEW

**Review by Ch3LL — DISMISSED** [review]

**Review by krionbsd — DISMISSED** [review]

**Review by garethgreenaway — DISMISSED** [review]

**Comment by Ch3LL** [comment]
re-run full all

**Comment by bryceml** [comment]
https://jenkins.saltproject.io/job/pr-centosstream-8-x86_64-py3-pytest/job/PR-60141/4/testReport/tests.integration.cli.test_batch/BatchTest/test_batch_run_grains_targeting/

is the failing test.  Looks like it can't handle the space in `CentOS Stream`

**Comment by Ch3LL** [comment]
@bryceml are you looking into the failure or does another engineer need to investigate?

**Comment by bryceml** [comment]
it was assigned to @waynew to fix.  If it needs re-assigned that's fine with me.

**Review by waynew — DISMISSED** [review]
:rocket:

File: tests/integration/cli/test_batch.py [hunk 1]
```diff
@@ -58,7 +58,9 @@ def test_batch_run_grains_targeting(self):
 
         os_grain = os_grain.strip()
         cmd = self.run_salt(
-            '-C "G@os:{} and not localhost" -b 25% test.ping'.format(os_grain),
+            '-C "G@os:{} and not localhost" -b 25% test.ping'.format(
+                os_grain.replace(" ", "?")
```
Comment (waynew) [inline tests/integration/cli/test_batch.py:62]: @bryceml I added this fix to the test that should make things work. We use fnmatch for globbing, and `?` matches any character. While this isn't a comprehensive fix, I think it's good enough. I tried a bunch of other methods:

- quotes: `G@os:"CentOS Stream"`
- backslash:  `G@os:CentOS\ Stream`
- brackets: `G@os:CentOS[ ]Stream`

I probably tried a few others that I'm forgetting now, but I wasn't able to figure out a way to get Salt to recognize any of those as valid, giving me "invalid compound target"

Unless someone else has a better solution, this was the best tradeoff:

- this test isn't actually testing how the OS grain was configured, only that grain targeting works
- for this test to allow a false positive (i.e. pass when it should have failed) we would literally have to have two OS grains configured that differ only by a space vs non-space like CentOS Stream and CentOSxStream.

It's reasonable enough, and the test *should* pass now.

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**